x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/influencer_filters/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { InfluencerFilter } from './influencer_filter';
```

Comment:
`EntityFilter` is a better name for this file / folder / component I think, as the charts are using one of the partitioning entities (which are likely to be influencers too, but not necessarily).

Reply:
Updated here https://github.com/elastic/kibana/pull/94396/commits/c2fc3b88fc2948a55a2acdf3e558eb678c4e1272